sourcemash/frontend/static/js/views/feed_card_view.js

```diff
@@ -0,0 +1,37 @@
+Sourcemash.Views.FeedCardView = Backbone.View.extend({
+    template: JST['feed-card'],
+
+    initialize: function(options) {
+        this.listenTo(this.model, 'change', this.render);
+        this.render();
+    },
+
+    events: {
+        "click #subscribe-switch": 'subscribeFromSwitch',
+	},
+
+    subscribeFromSwitch: function() {
+        if (this.model.get('subscribed')) {
+            this.model.save({'subscribed': false}, {success: this.subscribed});
+            
+            mixpanel.track("Unsubscribed", { "Feed Title": this.model.get('title') })
```

Comment:
When we talked the other day, I think we decided that how they unsubscribe is not as relevant as how they subscribe. There is no "conversion" metric to unsubscribing like there is to subscribing.